nashfmt: index out of range panic

When formating this:

```sh
	choice <= (
                -find $dir+"/" -maxdepth 1 | 
                sed "s#.*/##" | 
                sort | 
                uniq | 
                -fzf --exact -q "^"+$query -1 -0 --inline-info --header "select file: "
       )
```

```
nash-fmt errors:
panic: runtime error: index out of range

goroutine 1 [running]:
panic(0x4c9ea0, 0x55fbc0)
	/home/i4k/projects/personal/go/src/runtime/panic.go:500 +0x1a0
github.com/NeowayLabs/nash/ast.(*PipeNode).multiString(0xc420018b40, 0x0, 0x4)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/node_fmt.go:375 +0x5ca
github.com/NeowayLabs/nash/ast.(*PipeNode).string(0xc420018b40, 0x8, 0xc4200b8840, 0x29)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/node_fmt.go:395 +0x141
github.com/NeowayLabs/nash/ast.(*ExecAssignNode).string(0xc420018c00, 0xc4200b8810, 0x1f, 0xc42000f560)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/node_fmt.go:202 +0x248
github.com/NeowayLabs/nash/ast.(*ExecAssignNode).String(0xc420018c00, 0xc4200b8840, 0x29)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/node_fmt.go:218 +0x2b
github.com/NeowayLabs/nash/ast.(*BlockNode).String(0xc4200b2d50, 0xc42000f080, 0x1f)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/node_fmt.go:107 +0xf1
github.com/NeowayLabs/nash/ast.(*Tree).String(0xc42000e560, 0xc42000f060, 0x1b)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/tree.go:35 +0x41
github.com/NeowayLabs/nash/ast.(*FnDeclNode).String(0xc42004e190, 0xc420086b40, 0x117)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/node_fmt.go:538 +0x148
github.com/NeowayLabs/nash/ast.(*BlockNode).String(0xc42000a3c0, 0x0, 0x0)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/node_fmt.go:107 +0xf1
github.com/NeowayLabs/nash/ast.(*Tree).String(0xc42000e9a0, 0x0, 0x0)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/tree.go:35 +0x41
main.main()
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/cmd/nashfmt/main.go:68 +0x1cb

```

--- a/ast/node_fmt.go
+++ b/ast/node_fmt.go
@@ -277,6 +277,10 @@ func (n *CommandNode) toStringParts() ([]string, int) {
 		content = append(content, line)
 	}
 
+	if len(content) == 0 {
+		content = append(content, n.name)
+	}
+
 	for i := 0; i < len(n.redirs); i++ {
 		rstr := n.redirs[i].String()
 		totalLen += len(rstr) + 1
@@ -372,6 +376,7 @@ func (n *PipeNode) multiString() string {
 
 	for i := 0; i < len(content); i++ {
 		cmdContent := content[i].content
+
 		cmdContent[0] = "\t" + cmdContent[0]
 		tabLen := (len(cmdContent[0]) + 7) / 8
 

--- a/parser/parse_fmt_test.go
+++ b/parser/parse_fmt_test.go
@@ -315,6 +315,29 @@ func TestFmtPipes(t *testing.T) {
 		>[2] /dev/null
 )`,
 		},
+		{
+			`choice <= (
+                -find $dir+"/" -maxdepth 1 |
+                sed "s#.*/##" |
+                sort |
+                uniq |
+                -fzf --exact -q "^"+$query -1 -0 --inline-info --header "select file: "
+       )`,
+			`choice <= (
+	-find $dir+"/"
+		-maxdepth
+		1 |
+	sed "s#.*/##" |
+	sort |
+	uniq |
+	-fzf --exact
+		-q
+		"^"+$query -1
+		-0
+		--inline-info
+		--header "select file: "
+)`,
+		},
 	}
 
 	testFmtTable(testTable, t)

